lightning-background-processor/src/lib.rs

```diff
@@ -153,7 +153,7 @@ impl BackgroundProcessor {
 	/// functionality implemented by other handlers.
 	/// * [`NetGraphMsgHandler`] if given will update the [`NetworkGraph`] based on payment failures.
 	///
-	/// [top-level documentation]: Self
+	/// [top-level documentation]: BackgroundProcessor
```

Comment:
not really part of this PR, but whatever

Reply:
:man_shrugging: not worth a separate pr for it.